docs/API/Users_Guide/Data_Analysis.md

```diff
@@ -150,6 +153,379 @@ gene_start      gene_end        symbol  id
+
+An optional threshold (`min_median_log2_uqfpkm`) defines a minimum value for expression. Defaults to `1`.
+
+__Example 1__: A users wants to retrieve the most variably expressed gened for a given cohort id and all protein-coding genes.
+
+```Filter
+{
+    "cohort_id": "bcb99177-decb-437e-9614-df858789d6e2",
+    "gene_type": "protein_coding",
+    "selection_size": 10,
+    "min_median_log2_uqfpkm": 5
+}
+```
+
+```Shell
+curl -X 'POST' \
+  'https://api.gdc.cancer.gov/gene_expression/gene_selection' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "cohort_id": "bcb99177-decb-437e-9614-df858789d6e2",
+    "gene_type": "protein_coding",
+    "selection_size": 10,
+    "min_median_log2_uqfpkm": 5
+}'
+```
+
+```Response
+{
+    "gene_selection": [
+        {
+            "log2_uqfpkm_stddev": 4.199928035550041,
+            "log2_uqfpkm_median": 6.035571483832266,
+            "gene_id": "ENSG00000181617",
+            "symbol": "FDCSP"
+        },
+        {
+            "log2_uqfpkm_stddev": 3.3209310514900854,
+            "log2_uqfpkm_median": 8.878193845822231,
+            "gene_id": "ENSG00000186847",
+            "symbol": "KRT14"
+        },
+        {
+            "log2_uqfpkm_stddev": 3.121033211147152,
+            "log2_uqfpkm_median": 7.1985112055936735,
+            "gene_id": "ENSG00000163209",
+            "symbol": "SPRR3"
+        },
+        {
+            "log2_uqfpkm_stddev": 3.10197661282146,
+            "log2_uqfpkm_median": 9.804482756295696,
+            "gene_id": "ENSG00000171401",
+            "symbol": "KRT13"
+        },
+        {
+            "log2_uqfpkm_stddev": 3.0337841407700905,
+            "log2_uqfpkm_median": 8.164281801728123,
+            "gene_id": "ENSG00000124102",
+            "symbol": "PI3"
+        },
+        {
+            "log2_uqfpkm_stddev": 2.9335199502804015,
+            "log2_uqfpkm_median": 6.63085787272128,
+            "gene_id": "ENSG00000143556",
+            "symbol": "S100A7"
+        },
+        {
+            "log2_uqfpkm_stddev": 2.7164983400309297,
+            "log2_uqfpkm_median": 7.520801366683097,
+            "gene_id": "ENSG00000178372",
+            "symbol": "CALML5"
+        },
+        {
+            "log2_uqfpkm_stddev": 2.6568068682947668,
+            "log2_uqfpkm_median": 5.873762290074351,
+            "gene_id": "ENSG00000203785",
+            "symbol": "SPRR2E"
+        },
+        {
+            "log2_uqfpkm_stddev": 2.64389354922158,
+            "log2_uqfpkm_median": 10.99563880354997,
+            "gene_id": "ENSG00000143546",
+            "symbol": "S100A8"
+        },
+        {
+            "log2_uqfpkm_stddev": 2.600123088535626,
+            "log2_uqfpkm_median": 8.809619110422897,
+            "gene_id": "ENSG00000186832",
+            "symbol": "KRT16"
+        }
+    ]
+}
+```
+
+__Example 2__: A user wants to get the most variably expressed genes for a list of case UUIDs and a list of Ensemble gene id.
```

Comment:
I think Gene Ensembl[e] does not use the last `e`.

https://useast.ensembl.org/index.html

Reply:
s/gene id/gene ids

Reply:
changed for both comments, thank you!